### PR TITLE
[EDU-3169] feat: add automate secure - terraform

### PR DESCRIPTION
**Pull request by gabriel-azion**
https://aziontech.atlassian.net/jira/software/c/projects/EDU/boards/136?assignee=635c54ab548f1fe6f0c6ac38&selectedIssue=EDU-3169

### CODE REVIEW

File: src/content/docs/en/pages/automate-terraform.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Automate security with Azion Terraform provider
+description: Azion Terraform Provider is an open source project, registered in HashiCorp. Through the Azion SDK for Go, the provider communicates with the Azion APIs, so you can manage your infrastructure hosted on the the Azion platform, locally, as code.
+meta_tags: edge, security
+namespace: documentation_secure_automate_terraform
+menu_namespace: devtoolsMenu
```
Comment (hannah-sk) [inline src/content/docs/en/pages/automate-terraform.mdx:6]:
```suggestion
menu_namespace: secureMenu
```

File: src/content/docs/en/pages/automate-terraform.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+permalink: /documentation/products/secure/automate/terraform/
+---
+
+[Azion Terraform Provider](https://registry.terraform.io/providers/aziontech/azion/latest/docs) is a robust tool that integrates seamlessly with Terraform, the industry-leading Infrastructure as Code (IaC) platform.
+
+## Security with Terraform
+
+In modern web, security is key. Terraform addresses this concern by codifying infrastructure. This approach allows security configurations to be defined, versioned, and audited in tandem with application code. This ensures uniformity across different environments 
+and significantly reduces the risk of misconfigurations.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/automate-terraform.mdx:15]:
```suggestion
In modern web, security is key. Terraform addresses this concern by codifying infrastructure. This approach allows security configurations to be defined, versioned, and audited in tandem with application code, ensuring uniformity across different environments and significantly reducing the risk of misconfigurations.
```

File: src/content/docs/en/pages/automate-terraform.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Azion Terraform Provider and security
+
+| Scope | Resource | Data source |
+| --- | --- | --- |
+|  Edge Firewall Main Settings | [Managing main settings](https://registry.terraform.io/providers/aziontech/azion/latest/docs/resources/edge_firewall_main_setting) | [Consulting main settings](https://registry.terraform.io/providers/aziontech/azion/latest/docs/data-sources/edge_firewall_main_settings) |
+|  Edge Firewall Edge Functions Instances | [Managing edge functions instances](https://registry.terraform.io/providers/aziontech/azion/latest/docs/resources/edge_firewall_edge_functions_instance) | [Consulting edge functions instances](https://registry.terraform.io/providers/aziontech/azion/latest/docs/data-sources/edge_firewall_edge_function_instances) |
+|  Network list| [Managing network lists](https://registry.terraform.io/providers/aziontech/azion/latest/docs/resources/network_list) | [Consulting network lists](https://registry.terraform.io/providers/aziontech/azion/latest/docs/data-sources/network_lists) |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/automate-terraform.mdx:27]:
```suggestion
|  Network list | [Managing network lists](https://registry.terraform.io/providers/aziontech/azion/latest/docs/resources/network_list) | [Consulting network lists](https://registry.terraform.io/providers/aziontech/azion/latest/docs/data-sources/network_lists) |
```

**Review by hannah-sk — APPROVED** [review]
2 ajustes pequenos e uma sugestão!

File: src/content/docs/en/pages/automate-terraform.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: Automate security with Azion Terraform provider
+description: Azion Terraform Provider is an open source project, registered in HashiCorp. Through the Azion SDK for Go, the provider communicates with the Azion APIs, so you can manage your infrastructure hosted on the the Azion platform, locally, as code.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/automate-terraform.mdx:3]:
```suggestion
description: Azion Terraform Provider is an open source project, registered in HashiCorp. Through the Azion SDK for Go, the provider communicates with the Azion APIs, so you can manage your infrastructure hosted on the Azion's platform, locally, as code.
```

**Review by MarianaAguilera — APPROVED** [review]
Approved with @hannah-sk 's comments.

**Pull request merged**